WebViewControl/WebView.CefDragHandler.cs

```diff
@@ -22,11 +22,19 @@ private class CefDragHandler : IDragHandler {
                     }
                 }
 
+                var textDragging = OwnerWebView.TextDragging;
```

Comment:
why is this needed? I remember that that we had to do this for files because there was no way to get the filenames... but text could be obtain via js or is there any limitation?

Reply:
right. Text selection being dragged can be obtain via js only ondrop - which means we would only have knowledge of the text content being dragged when dropping in the target.
With this we allow the handling of such drags and its potential use (to set the object being dragged asap, for instance)

Reply:
Ok I get it.